index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Build a Survey Form</h1>
+
+    <p id="description">This is a short explanation</p>
+    
+    <form action="" id="survey-form">
+      <div>
+        <label id="name-label">
+          Name:
+        </label>
+        <input id="name" type="text" name="name" required placeholder="Joaquin Samaniego">
+        <label id="email-label">
+          Email:
+        </label>
+        <input id="email" type="email" name="email" required placeholder="axxxx@hotmail.com">
+        <label id="number-label">
+          Number:
+        </label>
+        <input id="number" type="number" min="0" max="100" placeholder="{x|x >= 0 v x =100}">
+      </div>
+      
+      <label class="selectionFit">
+        Select The best fit for you!
+      </label>
+      
+      <select name="" id="dropdown">
+        <option>Option one</option>
+        <option>Option two</option>
+      </select>
+      <div class="radio_botton space">
+        <div class="radio_botton">
+          <label id="radio1-label" for="option1">
+            Option 1
+          </label>
+          <input type="radio" name="radio" value="one" id="option1">
+        </div>
+        <div class="radio_botton">
+          <label id="radio2-label" for="option2">
+            Option 2
+          </label>
+          <input type="radio" name="radio" value="two" id="option2">
+        </div>
+      </div>
+      
+      <div class="check_box space">
```

Comment:
Good implementation, but you could use ul li blocks for list elements

Reply:
Overall, your communication has been polite and supportive, which is commendable. However, there are a few areas where we can further improve our feedback approach:

- [ ] Phrasing: When providing suggestions, it's important to frame them as polite suggestions rather than orders. For instance, instead of saying "Use semantic tags," you can say "I think it would be better to use semantic tags." This maintains a respectful and collaborative tone.

- [ ] Explanation: Including an explanation or a "why" for each requested change is crucial. It helps the developer understand the reasoning behind the suggestion and promotes learning. For example, when recommending the use of an article tag instead of a div, you can explain how it would improve the HTML structure and accessibility.

Thank you for your efforts in reviewing the code and offering valuable suggestions.